testrequirements/example_06_prequirements.preq

```diff
@@ -0,0 +1,6 @@
+global:
```

Comment:
Does it support this:

```
global:
  iops: "[0;500]k"
  bwps: "[80;200]MiB"
  lat:  "[20;50]usec" 

testcases:
  xnvme_fioe:
    iops: "[0;800]k"
```

Reply:
That is, the global will be the default to compare all testcases to, then when "testcases" is provided then for specific testcases the metrics are overwritten.

Reply:
Not yet - forgot to look at that after our chat yesterday. But it's definitely a nice feature. It shouldn't be difficult to add

Reply:
Would be great!

Reply:
Fixed